x-pack/plugins/cases/server/routes/api/internal/find_assignees.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { UserProfileService } from '../../../services';
+import { INTERNAL_FIND_ASSIGNEES_URL } from '../../../../common/constants';
+import { createCaseError } from '../../../common/error';
+import { createCasesRoute } from '../create_cases_route';
+
+export const findAssigneesRoute = (userProfileService: UserProfileService) =>
+  createCasesRoute({
+    method: 'get',
+    path: INTERNAL_FIND_ASSIGNEES_URL,
+    routerOptions: {
+      tags: ['access:casesFindAssignedUsers'],
+    },
+    params: {
+      query: schema.object({
+        owners: schema.arrayOf(schema.string()),
+        searchTerm: schema.string(),
+        size: schema.maybe(schema.number()),
+      }),
+    },
+    handler: async ({ request, response, context }) => {
```

Comment:
Is this only returning users who have a case assigned to them? I think it can be useful to allow a search on a user who has nothing currently assigned to them. Here for example I wanted to check if I had any cases assigned to the user who I am currently logged in as:

<img width="518" alt="image" src="https://user-images.githubusercontent.com/7405507/188411320-985a97a7-6205-4472-a282-73a65ad0d0e1.png">

GitHub seems to let you run a search on a user who has no issues / PRs assigned to them in the current repo.

Reply:
@arisonl @shanisagiv1 @MichaelMarcialis What is your opinion on this? In this PR we search only against users that are at least assigned to a case and not to all available users (assigned or not).

Reply:
We discussed it offline and we decided to show all users assigned or not.